hazelcast-jet-all/pom.xml

```diff
@@ -82,6 +82,64 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>unshaded</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <version>${maven.jar.plugin.version}</version>
+                        <configuration>
+                            <archive>
+                                <index>true</index>
+                                <compress>true</compress>
+                                <manifest>
+                                    <mainClass>com.hazelcast.jet.server.StartServer</mainClass>
+                                </manifest>
+                            </archive>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I think you only need this part here in the profile, since the rest is identical to the original